Core/Services/Auth/PasswordResetService.cs

```diff
@@ -11,15 +11,14 @@
 
 namespace Core.Services.Auth
 {
-    class PasswordResetService :  IPasswordResetService
+    class PasswordResetService : AuthServicesProvider, IPasswordResetService
```

Comment:
Is the default internal access modifier correct for this class?

Reply:
Yes, because I use this class only in this folder